manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/processor/PluginProcessor.java

```diff
@@ -96,6 +106,16 @@
                 WorkflowLogger.endActivity(Status.FAILED);
             }
         });
+
+        Map<ManifestMeta,String> manifestMetaVsSnippets = customSnippetsProvider.fetchUnmergedCustomSnippets();
+        if(manifestMetaVsSnippets != null && !manifestMetaVsSnippets.isEmpty()){
+            Set<String> kinds = new HashSet<>();
+            manifestMetaVsSnippets.forEach((manifestMeta, snippet)->{
+                kinds.add(manifestMeta.getKind());
+            });
+            String ignoreKindsForK8sPatches = String.join(",", kinds);
+            WorkflowLogger.warn(ManifestGeneratorActivity.IGNORING_CUSTOM_SNIPPET,ignoreKindsForK8sPatches);
+        }
```

Comment:
Extract this out in a new method
Fetching kinds Can be combined into a single operation.
kinds = manifestMetaVsSnippets.keySet().stream().map(ManifestMeta::getKind).collect(Collectors.toSet());